### PR TITLE
fix(lazy): suggested fixes for pref/lazyloading branch

### PR DESCRIPTION
**Pull request by pr-313**
### Description

Adds some edits suggested in the comments for PR #(3750)

### CODE REVIEW

File: lua/lvim/plugins.lua [hunk 1]
```diff
@@ -158,8 +158,10 @@ local core_plugins = {
     config = function()
       require("lvim.core.which-key").setup()
     end,
-    event = "VeryLazy",
+    event = {"User FileOpened"},
+    keys = {"<leader>"},
     enabled = lvim.builtin.which_key.active,
+    dependencies={"Comment.nvim"}
```
Comment (LostNeophyte) [inline lua/lvim/plugins.lua:164]:
```suggestion
```
that's not a good solution, comment.nvim should load when opening a file, it that doesn't work then the autocmd needs fixing

File: lua/lvim/plugins.lua [hunk 2]
```diff
@@ -158,8 +158,10 @@ local core_plugins = {
     config = function()
       require("lvim.core.which-key").setup()
     end,
-    event = "VeryLazy",
+    event = {"User FileOpened"},
+    keys = {"<leader>"},
```
Comment (LostNeophyte) [inline lua/lvim/plugins.lua:162]:
```suggestion
    event = "VeryLazy",
```
whichkey should be always loaded on startup, it works for more keys than just leader

**Review by LostNeophyte — CHANGES_REQUESTED** [review]

**Comment by LostNeophyte** [comment]
Thank you!

**Comment by pr-313** [comment]
@LostNeophyte 
Thank you for your feedback! 
I added a commit to account for your suggestions.
And apologies for the repeated force commits... it's my first time contributing :D

**Pull request merged**